sdk/identity/azure-identity/test/perf/inc/azure/identity/test/environment_credential_test.hpp

```diff
@@ -34,7 +35,8 @@ namespace Azure { namespace Identity { namespace Test {
      */
     void Setup() override
     {
-      m_tokenRequestContext.Scopes.push_back(m_options.GetMandatoryOption<std::string>("Scope"));
+      m_tokenRequestContext.Scopes.push_back(
+          m_options.GetOptionOrDefault<std::string>("Scope", "https://attest.azure.net/.default"));
```

Comment:
I thought that attest.azure.net was the endpoint for the attestation service, is it broader scope than that?